logrotation/test/py-utils/python_requirements.txt

```diff
@@ -0,0 +1,34 @@
+aiohttp==3.6.1
```

Comment:
why this separate requirements file now? Do we need something beyond supplied by cortx-prereq pkg supplied by RE? If we need to add new pkg, need strong justification and approval from Ujjwal over mail, then only RE would add in pre-req pkg

Reply:
It is for my testing, if we want I can remove the whole test folder from the PR, but the experiment alone won't work without that.